Package.swift

```diff
@@ -8,12 +8,11 @@ let package = Package(
     products: [
         .executable(name: "iblinter", targets: ["iblinter"]),
         .library(name: "IBLinterKit", targets: ["IBLinterKit"]),
-        .library(name: "IBLinterCore", targets: ["IBLinterCore"]),
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/drmohundro/SWXMLHash.git", from: "4.0.0"),
+        .package(url: "https://github.com/IBDecodable/IBDecodable.git", .branch("master")),
```

Comment:
I think this is fine for now but we should definitely use a specific version in the future